### ISSUE TITLE
Enable pg_stat_statements?

### ISSUE THREAD

**Issue by edbizarro** [ANXS/postgresql#370]
There is a way to enable pg_stat_statements? This extension required an extra config `pg_stat_statements.track = all` but I can't find a way to do this in the yml

**Comment by gclough** [comment]
You will need to enable it in `shared_preload_libraries`:

```
  postgresql_shared_preload_libraries:
    - "pg_stat_statements"
```

And then add it as an extension:

```
  postgresql_database_extensions:
    - db: postgres
      extensions:
        - pg_stat_statements
```

If you're wanting to add extra parameters to the postgresql.conf... I'm not sure there is a way to do that.  It would need to be added to the `templates/postgresql.conf-{{ postgresql_version }}.j2` file, which is the source for all postgresql.conf settings.

**Comment by gclough** [comment]
This will get fixed by #277

**Comment by attenzione** [comment]
Would be nice to see it in a release. #277 is open too long 😢
